packages/experimental/Shimmer/src/Shimmer.styling.ts

```diff
@@ -5,13 +5,10 @@ import { shimmerName, ShimmerProps, ShimmerSlotProps, ShimmerTokens } from './Sh
 export const stylingSettings: UseStylingOptions<ShimmerProps, ShimmerSlotProps, ShimmerTokens> = {
   tokens: [
     () => ({
-      toValue: 30,
       duration: 7000,
       delay: 0,
-      gradientTintColor: 'white', // Baseline component is the light mode color
-      shimmerTintColor: '#E1E1E1', // Baseline component is the light mode color
-      width: 200,
-      height: 100,
+      shimmerWaveColor: 'white', // Baseline component is the light mode color
```

Comment:
If we have these colors in the global tokens it might be good to convert to pull from those so we have less hardcoded colors in our repo.

Reply:
yeah I need to check in w/ lineage on their design figma; these may be existing tokens (e.g. I use bodyDivider for one) or need to be new

Reply:
I think the original hex codes came from the older FluentUI Apple palette. #E1E1E1 maps to a few things from there. I agree that a followup should be to use the global token, especially if we can get that without a theme reference